Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.9-slim-buster AS compile-image
+FROM python:3.10.14-slim-bookworm AS compile-image
```

Comment:
Thanks @gamboz.
Now that you are updating it, could it be `3.11.9` instead of `3.10.14`?